packages/jupyter-ai/jupyter_ai/chat_handlers/clear.py

```diff
@@ -1,5 +1,6 @@
 from typing import List
 
+from jupyter_ai.chat_handlers.help import build_help_message
```

Comment:
I think this is left over from another branch. Can you please revert?

Reply:
Oh yes, will do! Thanks

Reply:
@JasonWeill Done